### DISCUSSION TITLE
Can't add Gif

### DISCUSSION THREAD

**Discussion by Aveiro11** [academicpages/academicpages.github.io#2154] [Q&A]
Hello everyone. How do i add gif in the adbout.md section of pages. I tried adding gif here, it shows in my github preview but dosent load in my site when i go to my .io

**Comment by rjzupkoii** [comment]
The pathing for images can get to be a bit confusing at time, [this answer should help](https://github.com/academicpages/academicpages.github.io/discussions/1990#discussioncomment-8831790).
